components/SingleRatingsTable/SingleGoalieRatingsTable.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import { GoalieRatings } from '../..';
+
+import SingleRatingsTable from '.';
+
+interface Props {
+  data: Array<GoalieRatings>;
+  pagination?: boolean;
+  teamPage?: boolean;
+  searching?: boolean;
+}
+
+function SingleGoalieRatingsTable({
+  data: players,
+  pagination = false,
+  teamPage = false,
+  searching = false,
+}: Props): JSX.Element {
+  const columnData = [
+    {
+      Header: 'Player Info',
+      id: 'player-table-basic-info',
+      columns: [
+        {
+          Header: 'Season',
+          id: 'player-table-season',
+          accessor: 'season',
+          // Create cell which contains link to player
```

Comment:
If this is for individual players I don't see why we should be linking at all. If I am not understanding this file properly let me know

Reply:
this file is where the link takes the user to.  This table points to an index table without the links at all.

Reply:
the comment is deceptive, I will remove it